butterknife/src/main/java/butterknife/internal/SupportV4Utils.java

```diff
@@ -0,0 +1,42 @@
+package butterknife.internal;
+
+import android.content.res.Resources;
+import android.graphics.drawable.Drawable;
+import android.support.v4.graphics.drawable.DrawableCompat;
+import android.util.TypedValue;
+
+import static butterknife.internal.CompatUtils.getColor;
+import static butterknife.internal.CompatUtils.getDrawable;
+
+/** Requires v4 Support Library. Check {@link #HAS_SUPPORT_V4_LIBRARY} before use. */
+final class SupportV4Utils {
+  static final boolean HAS_SUPPORT_V4_LIBRARY = hasSupportV4LibraryOnClasspath();
+
+  private SupportV4Utils() {
+    throw new AssertionError("No instances.");
+  }
+
+  static Drawable setDrawableTint(Resources res, Resources.Theme theme, int drawableId,
+      int attributeId) {
+    TypedValue outValue = new TypedValue();
+    boolean attributeFound = theme.resolveAttribute(attributeId, outValue, true);
+    if (!attributeFound) {
+      throw new Resources.NotFoundException("Tint attribute not found.");
```

Comment:
This exception should include both the attribute ID and do a reverse-lookup on the attribute name. Look at the `findRequiredView` for an example.